applications/nublado/templates/purger-cronjob.yaml

```
@@ -0,0 +1,65 @@
+{{if .Values.purger.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: purger
+spec:
+  schedule: {{ .Values.purger.schedule | quote }}
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          automountServiceAccountToken: false
+          {{- with .Values.purger.tolerations }}
+          tolerations:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.purger.affinity }}
+          affinity:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          containers:
+          - name: purger
+            image: "{{ .Values.purger.image.repository }}:{{ .Values.purger.image.tag | default .Chart.AppVersion }}"
+            imagePullPolicy: "{{ .Values.purger.image.pullPolicy }}"
+            {{- with .Values.purger.resources }}
+            resources:
+              {{- toYaml . | nindent 14 }}
+            {{- end }}
+            {{- if .Values.controller.slackAlerts }}
```

If we want to use the slackAlerts from the purger I think it would be:

```suggestion
            {{- if .Values.purger.slackAlerts }}
```

---

Let's just remove it from the purger as a separate config.  If the controller sends slack alerts so should the purger.